Pod install fails for versions after 1.0.3 due to VersionChecker.swift

## Bug report

Installing Apollo via Cocoapods for versions after 1.0.4+ fails due to an issue with a file `VersionChecker.swift`. I checked the commits and it looks like this file was added 8 days ago before v1.0.4 was released. Installing a previous ver worked fine.

This is the error message:

```
/Users/y/Library/Caches/CocoaPods/Pods/Release/Apollo/1.0.4-7c977/Sources/CodegenCLI/Extensions/VersionChecker.swift:34:29: error: variable binding in a condition requires an initializer
      if let projectRootURL {
                            ^
/Users/y/Library/Caches/CocoaPods/Pods/Release/Apollo/1.0.4-7c977/Sources/CodegenCLI/Extensions/VersionChecker.swift:52:29: error: variable binding in a condition requires an initializer
      if let projectRootURL {
                            ^
make: *** [build-cli-for-cocoapods] Error 1
```

## Versions

Please fill in the versions you're currently using: 

- `apollo-ios` SDK version: 1.0.4+
- Xcode version: 13.4
- Swift version: 5.6
- Package manager: Cocoapods

## Steps to reproduce

Install Apollo 1.0.4 or 1.0.5 using Cocoapods

--- a/Package.swift
+++ b/Package.swift
@@ -1,4 +1,4 @@
-// swift-tools-version:5.6
+// swift-tools-version:5.7
 // The swift-tools-version declares the minimum version of Swift required to build this package.
 
 import PackageDescription

--- a/Sources/ApolloCodegenLib/Templates/SwiftPackageManagerModuleTemplate.swift
+++ b/Sources/ApolloCodegenLib/Templates/SwiftPackageManagerModuleTemplate.swift
@@ -16,7 +16,7 @@ struct SwiftPackageManagerModuleTemplate: TemplateRenderer {
     let casedSchemaName = config.schemaName.firstUppercased
 
     return TemplateString("""
-    // swift-tools-version:5.6
+    // swift-tools-version:5.7
 
     import PackageDescription
 

--- a/Tests/ApolloCodegenTests/CodeGeneration/Templates/SwiftPackageManagerModuleTemplateTests.swift
+++ b/Tests/ApolloCodegenTests/CodeGeneration/Templates/SwiftPackageManagerModuleTemplateTests.swift
@@ -33,7 +33,7 @@ class SwiftPackageManagerModuleTemplateTests: XCTestCase {
     buildSubject()
 
     let expected = """
-    // swift-tools-version:5.6
+    // swift-tools-version:5.7
     """
 
     // when

